source/partials/_nav.slim

```diff
@@ -5,20 +5,21 @@
   = nav_overlay_item('Apprenticeship', '/apprenticeship')
   = nav_overlay_item('Blog', '/blog')
 
-nav#main-nav.Nav.Nav--topPositioned class="#{current_page.data.nav_class}" aria-label=('Main navigation')
-  = link_to '/', class: 'Nav-logo'
-    .NavLogo class=nav_logo_class
+header
+  #main-nav.Nav.Nav--topPositioned class="#{current_page.data.nav_class}"
+    = link_to '/', "aria-label" => "Homepage", class: 'Nav-logo'
+      .NavLogo class=nav_logo_class
 
-  .Nav-menu
-    .Burger class=nav_burger_class
-      span.Burger-line
-      span.Burger-line
-      span.Burger-line
+    .Nav-menu
+      .Burger class=nav_burger_class
+        span.Burger-line
+        span.Burger-line
+        span.Burger-line
 
-  ul.Nav-items
-    = nav_item('Work', '/work')
-    = nav_item('Company', '/company')
-    = nav_item('Community', '/community')
-    = nav_item('Apprenticeship', '/apprenticeship')
-    = nav_item('Blog', '/blog')
-.u-navPlaceholder
+    nav aria-label='Main navigation' role='navigation'
```

Comment:
I'm not a fan of having this element without a `Nav-*` class.
Couldn't this have the `.Nav-items` class? One thing I've been wondering and researched one time, is that a nav doesn't require its items or anything inside to be a `ul`. Is that *list* really necessary?